src/ammo.cpp

```diff
@@ -64,7 +64,7 @@ void ammunition_type::check_consistency()
         const auto &at = ammo.second.default_ammotype_;
 
         // TODO: these ammo types should probably not have default ammo at all.
-        if( at == "UPS" || at == "components" || at == "thrown" ) {
+        if( at == "UPS" || at == "UPS_AIR" || at == "components" || at == "thrown" ) {
```

Comment:
`UPS_AIR` is a really bad name for this. Change it to so it doesn't mention UPS as it doesn't have any relationship to UPS at all.

Reply:
Like I said it's a compressed air power supply. which is a thing. Have you read the link I posted? And it is universal in that it is fits all devices currently requiring external compressed air to function.

Reply:
It is not a compressed air power supply, that's something different.
Change the name.

Reply:
OK I think I whacked all the moles. Uploading now.